ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -165,6 +213,45 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-ipi-proxy-cco-manual-security-token-service
+- as: aws-ipi-sdn-arm-f28
+  cron: 4 9 3 * *
+  steps:
```

Comment:
I think sdn network is the default network in OCP 4.14. so we don't need to add this job.
@zhaozhanqi @jianlinliu Please correct me if I am wrong.

Reply:
4.14 default network cni is OVN

Reply:
@dis016 we still need aws-ipi-sdn-arm, so please add this back, thanks.

Reply:
added back